src/main/java/school/faang/user_service/controller/recommendation/RecommendationRequestController.java

```diff
@@ -0,0 +1,46 @@
+package school.faang.user_service.controller.recommendation;
+
+import lombok.RequiredArgsConstructor;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.FieldError;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import school.faang.user_service.dto.recommendation.RecommendationRequestDto;
+import school.faang.user_service.service.recommendation.RecommendationRequestService;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/recommendation")
+public class RecommendationRequestController {
+    private final RecommendationRequestService recommendationRequestService;
+
+    @PostMapping()
+    public RecommendationRequestDto requestRecommendation(@Validated @RequestBody RecommendationRequestDto recommendationRequestDto) {
+        return recommendationRequestService.create(recommendationRequestDto);
+    }
+
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
```

Comment:
exception handler should be moved to separete handler(create if not exist) package